packages/terra-clinical-data-grid/tests/jest/__snapshots__/DataGrid.test.jsx.snap

```diff
@@ -1933,33 +1933,6 @@ exports[`DataGrid Snapshots should render a DataGrid with selectable rows, colum
 
 exports[`getDerivedStateFromProps correctly applies the theme context className 1`] = `
 <ThemeContextProvider
-  intl={
```

Comment:
Are such intl objects no longer needed, or are they still used but moved elsewhere?

Reply:
The data grid [jests](https://github.com/cerner/terra-clinical/blob/fdd35b3a0dd7d4d7c443670d68488b776b69be22/packages/terra-clinical-data-grid/tests/jest/DataGrid.test.jsx#L388) are written using WrappedComponent. So the Data Grid is rendered without the injectIntl HOC in the snapshots.